index.html

```diff
@@ -1 +1 @@
-My Web Page
+<h6>My Web Page</h6>
```

Comment:
Hey this is how you make the smallest heading! You can create the largest heading instead with:
```
<h1>My Web Page</h1>
```